topology/University of Connecticut/UConn-HPC/UConn-HPC.yaml

```diff
@@ -0,0 +1,100 @@
+Production: true
+SupportCenter: GLUEX
+GroupDescription: OSG partition of the Storrs HPC Cluster
+GroupID: 1092
+Resources:
+  UConn-HPC_squid:
+    Active: true
+    Description: squid server for UConn-HPC
+    ID: 1093
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          ID: 650bf84a5cf49caf504fed22ce07b98580c6fa12
+          Name: Richard T. Jones
+      Miscellaneous Contact:
+        Primary:
+          ID: 650bf84a5cf49caf504fed22ce07b98580c6fa12
+          Name: Richard T. Jones
+      Resource Report Contact:
+        Primary:
+          ID: 650bf84a5cf49caf504fed22ce07b98580c6fa12
+          Name: Richard T. Jones
+      Security Contact:
+        Primary:
+          ID: 650bf84a5cf49caf504fed22ce07b98580c6fa12
+          Name: Richard T. Jones
+    FQDN: osgce.storrs.hpc.uconn.edu
+    FQDNAliases:
+      - cn410.storrs.hpc.uconn.edu
+    DN: /DC=org/DC=incommon/C=US/ST=Connecticut/L=Storrs/O=University of Connecticut/OU=Academic IT/CN=cn410.storrs.hpc.uconn.edu
+    Services:
+      Squid:
+        Description: squid server for Storrs HPC cluster
+    Tags:
+      - CC*
```

Comment:
@rjones30 is Storrs a resource using funds from a Campus Cyberinfrastructure grant?